content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,293 @@
+| **HTTP Proxy(--http_proxy)**| Leave this blank unless you are using an HTTP Proxy. This setting will be propagated to all the nodes launched in the proxy network. Example: ``http://USERNAME:PASSWORD@PROXYIP:PROXYPORT``.|
+| **No Proxy(--no_proxy)**| The default is blank. You can add a comma-separated list of local network classless inter-domain routing (CIDR) addresses, hostnames, and domain names that should be excluded from being a proxy. This setting will be propagated to all the nodes to bypass the proxy server. Example: ``maas.company.com,10.10.0.0/16``.|
+| **Pod CIDR (--pod_cidr)**|Enter the CIDR pool that will be used to assign IP addresses to pods in the cluster. The pod IP addresses should be unique and should not overlap with any virtual machine IPs in the environment.|
+| **Service IP Range (--svc_ip_range)**|Enter the IP address range that will be used to assign IP addresses to services in Kubernetes clusters. The service IP addresses should be unique and not overlap with any virtual machine IPs in the environment.|
+
+<br />
+
+
+#### MAAS Account Information
+
+<WarningBox>
+
+When specifying the MAAS endpoint, do not add a trailing `/` as it can result in DNS resolution errors.
+
+</WarningBox>
+
+|**Parameter**| **Description**|
+|-------------|----------------|
+| **API Endpoint** |Enter the MAAS API endpoint without a trailing /. This can be a domain or IP address. Example: ``http://10.11.12.13:5240/MAAS.``|
+| **API Key** |Enter an active MAAS API key to use for authentication.|
+
+<br />
+
+7. When the installer prompts you, select the following to configure the MAAS server:
+
+    - Availability Zone
+    - Domain
+    - Resource Pool
+    - One node (no HA) or three nodes (HA)
+
+    <br />
+
+    The installer generates a ``pcg.yaml`` configuration file and displays the file location on the console. For example: <br />
+
+``Config created:/opt/spectrocloud//User-defined-MaaS-Gateway-Name-20210805155034/pcg.yaml``
+
+<br />
+
+8. Copy the ``pcg.yaml`` file out of ``/tmp/install-user-defined-MaaS-Gateway_Name-20210805155034/pcg.yaml`` and into ``/tmp``.
+
+``` bash
+cp /tmp/install-user-defined-MaaS-Gateway_Name-20210805155034/pcg.yaml /tmp
+```
+<br />
+
+<WarningBox>
+
+After you copy ``pcg.yaml``, delete it from ``/tmp/install-user-defined-MaaS-Gateway_Name/pcg.yaml`` because it contains your Spectro Cloud credential.
+
+</WarningBox>
+
+<br />
+
+9. To deploy the gateway, copy the following code snippet to your terminal and provide the gateway configuration file as input.
+<br />
+
+    ```bash
+    docker run -it --rm \
+    --net=host \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /tmp:/opt/spectrocloud \
+    gcr.io/spectro-images-public/release/spectro-installer:1.0.12 \
+    -s true \
+    -c //opt/spectrocloud/pcg.yaml
+    ```
```

Comment:
This can all be removed as we are doing the install in one swing.

Reply:
Removed.